CONTRIBUTING.md

```diff
@@ -242,25 +241,6 @@ A problem must have a unique slug. This slug is used as
   Reference the PR in x-common if it hasn't been merged yet,
   this **must not** be merged until the exercism/x-common PR is merged.
 
-## Improving Consistency By Extracting Shared Test Data
```

Comment:
Not entirely covered by https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/improving-consistency-across-tracks.md